### PR TITLE
add url rest test

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,26 +34,26 @@
...
         "lodash": "^4.17.21",
         "prettier": "^1.19.1",
         "source-map-support": "^0.5.16",
-        "ts-morph": "^9.0.0",
-        "fs-extra": "^10.0.0"
+        "ts-morph": "^9.0.0"
     },
     "devDependencies": {
         "@autorest/test-server": "^3.0.27",
+        "@azure-rest/core-client": "1.0.0-beta.8",
```
Comment (joheredi) [inline package.json:56]: Why are we moving this to devDependencies?

**Review by joheredi — APPROVED** [review]
Looks great! :shipit:

File: package.json (hunk 1 above)
Comment (qiaozha) [reply package.json:56]: It's a mistake, let me change it back

**Comment by joheredi** [comment]
I think you'll need to run `npm run test` and `npm run smoke-test` to get the changes in the generated test files. Let's review the changes and make sure nothing unintended is changing

**Comment by qiaozha** [comment]
finally get it to be green :)

**Pull request merged**